server/lib/src/idm/ldap.rs

```diff
@@ -445,18 +445,36 @@ impl LdapServer {
                 e
             })?
         };
-
-        let lae = LdapAuthEvent::from_parts(target_uuid, pw.to_string())?;
-        idm_auth.auth_ldap(&lae, ct).await.and_then(|r| {
-            idm_auth.commit().map(|_| {
-                if r.is_some() {
-                    security_info!(%dn, "✅ LDAP Bind success");
-                } else {
-                    security_info!(%dn, "❌ LDAP Bind failure");
-                };
-                r
+        if idms
+            .proxy_read()
+            .await
+            .qs_read
+            .d_info
+            .d_ldap_allow_unix_pw_bind
+        {
+            debug!(
+                "here {:?}",
+                idms.proxy_read()
+                    .await
+                    .qs_read
+                    .d_info
+                    .d_ldap_allow_unix_pw_bind
+            );
+            let lae = LdapAuthEvent::from_parts(target_uuid, pw.to_string())?;
+            idm_auth.auth_ldap(&lae, ct).await.and_then(|r| {
```

Comment:
Why not read this setting for allowing the unique bind here inside of auth_ldap? I think because youi have the proxy read open above that's causing a double-read transaction which is wrong. There should only be *one* transaction per operation. So I'd revert the changes just here, and look into checking this flag in auth_ldap.